recipe/mpiexec.sh

```diff
@@ -9,6 +9,7 @@ fi
 if command -v "${PREFIX}/bin/ompi_info" >/dev/null; then
     export OMPI_MCA_plm=isolated
     export OMPI_MCA_rmaps_base_oversubscribe=yes
+    export OMPI_MCA_btl=tcp,self
     export OMPI_MCA_btl_vader_single_copy_mechanism=none
```

Comment:
Then this becomes most likely irrelevant, as Open MPI will no longer attempt to use `btl=vader` if that is ever available. Anyway, probably not worth fixing.

Reply:
Good to know, thanks!